xml/System.Windows.Forms/ToolStripItem.xml

```diff
@@ -2464,9 +2465,12 @@
   
     -   If <xref:System.Windows.Forms.ToolStrip.AutoSize%2A> is `false` and <xref:System.Windows.Forms.ToolStripItemImageScaling> is <xref:System.Windows.Forms.ToolStripItemImageScaling.None>, neither image nor <xref:System.Windows.Forms.ToolStrip> scaling occurs.  
   
+If the <xref:System.Windows.Forms.ToolStripItem.ImageList%2A> property value is changed to `null`, the <xref:System.Windows.Forms.ToolStripItem.ImageIndex%2A> property returns its default value, -1. However, the assigned <xref:System.Windows.Forms.ToolStripItem.ImageIndex%2A> value is retained internally and used when another <xref:System.Windows.Forms.ImageList> object is assigned to the <xref:System.Windows.Forms.ToolStripItem.ImageList%2A> property. If the new <xref:System.Windows.Forms.ImageList> assigned to the <xref:System.Windows.Forms.ToolStripItem.ImageList%2A> property has an <xref:System.Windows.Forms.ImageList.ImageCollection.Count%2A?displayProperty=nameWithType> property value that is less than or equal to the value assigned to the <xref:System.Windows.Forms.ToolStripItem.ImageIndex%2A> property minus one (to account for the collection being a zero-based index), the <xref:System.Windows.Forms.ToolStripItem.ImageIndex%2A> property value is adjusted to one less than the <xref:System.Windows.Forms.ImageList.ImageCollection.Count%2A> property value.
```

Comment:
Should `ToolStripItem.ImageList` be `ToolStrip.ImageList` instead? I'm not seeing that property on `ToolStripItem`.